tsdfileapi/api.py

```diff
@@ -531,6 +531,9 @@ def write_file(self, filemode, filename, filebody, tenant):
                 os.chmod(self.path_part, _RW_RW___)
             self.new_paths.append(self.path_part)
             if self.backend == 'sns':
+                if not os.path.lexists(tsd_hidden_folder):
```

Comment:
Will the following lines here ever run? A os.path.lexists() check runs as part of the sns_dir() call that assigns tsd_hidden_folder's value.

Reply:
ack.